recipes/ssake/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+sed -i.bak "s:/usr/bin/perl:/usr/bin/env perl:" ssake/SSAKE
+sed -i.bak "s:/usr/bin/perl:/usr/bin/env perl:" ssake/tools/*.pl
```

Comment:
Can you try to patch this upstream as well?

Reply:
Done :)